### DISCUSSION TITLE
Performance and playing with parameters

### DISCUSSION THREAD

**Discussion by tom221085** [miniscope/minian#137] [Q&A] [answered]
Hello!

I currently started to try out minian as an alternative to min1pipe since it doesn't require a fancy computer and I also had a lot of troubles regarding error messages with min1pipe. However, I extracted 184 ROIs using min1pip from a data set recorded by UCLA V4 miniscopes. I ran the same data set through the minian pipeline and only receive about 80 ROIs. I tried different parameters but basically always got the same results. I was wondering why is that, given the fact that the CNMF part of minian is similar to the min1pipe algorithm.

Can you help me with this issue or am I doing anything wrong here?

Thanks,
Thomas

**Comment by phildong** [comment]
Hey Thomas,

So there are lots of moving parts and many reasons this could happen. The first place I would start looking is during seed initialization step. Do you have lots of seeds to begin with? Knowing that would help narrow the problem down to motion-correction/pre-processing stage or CNMF stage. Also Minian usually prints some info when it's dropping units. Maybe those could be helpful too? Let me know of any updates!

**Reply by phildong** [reply]
p.s. we are trying out the new github discussion featuer to separate questions from bug reports and help the community better navigate these information. I hope you don't mind I converted your issue to a question!

**Comment by Yifang-Yuan** [comment, marked as answer]
Maybe try a smaller spatial sparse penalty, which might give you more ROIs which are reasonable, like 0.0001~0.0005(works for me).

**Reply by tom221085** [reply]
Hi phildong and Yifang-Yuan,
thanks for the advice. I tried using smaller spatial sparse penalty and it gives me more ROIs. Thank you very much!
